website/content/userguide/permanent-data.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: "Access permanent data"
+menuTitle: "Access permanent data"
+weight: 49
+---
+
+## Create the filesystem
+
+gokrazy provides a handy way to create the permanent data filesystem on the forth partition of your gokrazy installation.
+The [gokrazy/mkfs](https://github.com/gokrazy/mkfs) program will do all the required steps to create the filesystem.
+To actually access the permanent data partition from your own program, gokrazy will mount the partition under `/perm` directory during the startup.
+
+You can either include this program in your `gokr-packer` command line:
+
+```
+gokr-packer \
+  -overwrite=/dev/sdx \
+  -serial_console=disabled \
+  github.com/gokrazy/fbstatus \
+  github.com/gokrazy/hello \
+  github.com/gokrazy/serial-busybox \
+  github.com/gokrazy/mkfs
+```
+
+…or, if you want to run it only once without otherwise including it in your
+installation, you can use `gok run`:
+
+```
+git clone https://github.com/gokrazy/mkfs
+cd mkfs
+gok run -i bakery
+```
+
+```bash
+sudo mkfs.ext4 /dev/sdc
+```
+
+## Permanent data in action
+
+The Go program demonstrates the functionality of the permanent data mount:
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+)
+
+const config = "/perm/config.json"
```

Comment:
Let’s store this file in a subdirectory. Each gokrazy process uses a subdirectory after the process’s basename, e.g. `tailscaled` stores data in `/perm/tailscaled` by default (passed to the program via the `HOME` environment variable).

How about `/perm/my-example/config.json`?